packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -585,16 +582,15 @@ export class ConnectionFlow {
     try {
       const result = await client.query(Q('io.cozy.files').getById(folderId))
       folder = result.data
+      if (folder.path !== account.defaultFolderPath) {
+        account.defaultFolderPath = folder.path
+        const savedAccount = await saveAccount(client, konnector, account)
+        return savedAccount
+      }
```

Comment:
can you explain why we do that inside the try / catch (aka why did you move the call)?

Reply:
Just a better practice. To avoid to have uncaught errors and saveAccount can be a source of errors.